lib/ButtonNew/ButtonBase.js

```diff
@@ -26,7 +26,8 @@ function ButtonBase({
 
       'text-base py-3 px-4': size && size === sizes.md,
       'text-base p-2': size && size === sizes.sm,
-      'text-sm p-05 px-2 rounded-small': size && size === sizes.xs
+      'text-sm p-05 px-2 rounded-small': size && size === sizes.xs,
+      'pointer-events-none': disabled
```

Comment:
🔴  When a button is disabled, isn't it disabled so why do we need to do this? 🤔

Reply:
I mentioned it in the description, this needs to be here in order for tooltips to work on disabled buttons and something I have to add in quite often so makes sense for gui!

Reply:
The image captivated me and I missed the detail about the workaround 😄